docs/concepts/automations.md

```diff
@@ -391,7 +391,163 @@ To detect when the average lateness of your Kubernetes workloads (running on a w
+
+The `within` parameter for compound and sequence triggers iconstrains how close in time (in seconds) the child triggers must fire to satisfy the composite trigger.  For example,if the `daily-export-initiator` flow runs, but the other three flows don't write their result files until three hours later, this trigger won't fire.  Placing these time constraints on the triggers can prevent a misfire if you know that the events will generally happen within a specific timeframe, and you don't want a stray older event to be included in the evaluation of the trigger.  If this isn't a concern for you, you may omit the `within` period, in which case there is no limit to how far apart in time the underlying triggers occur.
+
+Any type of triggers may be composed into higher-order composite triggers, including proactive event triggers and metric triggers.  In this example, the trigger will fire if we see flow runs that are stuck in `Pending`, a work pool becoming unready, or the average amount of `Late` work in your workspace going over 10 minutes:
+
+```json
+{
+  "type": "compound",
+  "require": "any",
+  "triggers": [
+    {
+      "type": "event",
+      "posture": "Proactive",
+      "after": ["prefect.flow-run.Pending"],
+      "expect": ["prefect.flow-run.Running", "prefect.flow-run.Crashed"],
+      "for_each": ["prefect.resource.id"],
+      "match_related": {
+        "prefect.resource.name": "daily-customer-export",
+        "prefect.resource.role": "flow"
+      }
+    },
+    {
+      "type": "event",
+      "posture": "Reactive",
+      "expect": ["prefect.work-pool.not_ready"],
+      "match": {
+        "prefect.resource.name": "kubernetes-workers",
+      }
+    },
+    {
+      "type": "metric",
+      "metric": {
+        "name": "lateness",
+        "operator": ">",
+        "threshold": 600,
+        "range": 3600,
+        "firing_for": 300
+      }
+    }
+  ]
+}
+```
+
+For compound triggers, the `require` parameter may be `"any"`, `"all"`, or a number between 1 and the number of underlying triggers.  In the example above, if you feel that you are receiving too many spurious notifications for issues that resolve on their own, you can specify `{"require": 2}` to express that any **two** of the triggers must fire in order for the compound trigger to fire indicating a more serious problem.  Sequence triggers, on the other hand, always require all of their underlying triggers to fire.
+
+Compound triggers are defined as:
+
+| Name         | Type                        | Description                                                                  |
+| ------------ | --------------------------- | ---------------------------------------------------------------------------- |
+| **require**  | number, `"any"`, or `"all"` | How many of the underlying triggers must fire for this trigger to fire       |
+| **within**   | time, in seconds            | How close in time the underlying triggers must fire for this trigger to fire |
```

Comment:
just curious, have we standardized on using "underlying" vs like "child"? I feel like we've been internally calling them child triggers but I'm not sure if that's intuitive to users

Reply:
Yeah I wasn't sure about calling them "children" either, they're just called "`triggers`" in code.

Reply:
Yeah but we have the `CompoundTriggerChildFiring` table right, which when we bring events to open source will expose that table and naming convention to users, maybe?